src/patient-registration/section/identifier/identifier-section.component.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { IdentifierInput } from '../../input/custom-input/identifier/identifier-input.component';
+import { PatientIdentifierType } from '../../patient-registration-helper';
+import styles from './../section.css';
+
+export const IdentifierSection: React.FC<{ identifierTypes: PatientIdentifierType[] }> = ({ identifierTypes }) => {
```

Comment:
Would love to see `Array<PatientIdentifierType>` instead of `PatientIdentifierType[]`. In general - why not use `IdentifierSectionProps` here? Would make it easier to read imho.

Reply:
Same as above, the PatientIdentifierType was part of a different story (we merged back master into our branch) and we don't feel familiar enough with this story to make changes to the logic without understanding.

Reply:
This is not about logic, but about typing. There is no functional change here.

I only see it here as a change - so I'd appreciate it being corrected. Thanks!

Reply:
We refactored the typing like you suggested :)